test/e2e/framework/framework.go

```diff
@@ -156,3 +170,17 @@ func (f *Framework) CleanUp(t *testing.T, cleanupFunc func()) {
 		}
 	})
 }
+
+func (f *Framework) IsOpenshiftCluster() bool {
+	c, err := f.getOpenshiftClient()
```

Comment:
I think we might be better off to initialize this in https://github.com/rhobs/observability-operator/blob/62bcd73f3cce816269e038d7d87f74ba24aa42ae/test/e2e/main_test.go#L84 and rely on the client being there. Otherwise we have to remember to initialize this in future functions that needs the openshift client.